config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -399,6 +387,28 @@ public CorsSpec cors() {
 		return this.cors;
 	}
 
+	/**
+	 * @since 5.2.0
+	 * @author Ankur Pathak
+	 * Configures annonymous authentication
+	 *
+	 * <pre class="code">
+	 *  &#064;Bean
+	 *  public SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http) {
+	 *      http
+	 *          // ...
+	 *          .csrf().key(key)
```

Comment:
please fix this Javadoc

Reply:
This is fixed like this:
```
/**
	 * @since 5.2.0
	 * @author Ankur Pathak
	 * Configures annonymous authentication
	 *
	 * <pre class="code">
	 *  &#064;Bean
	 *  public SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http) {
	 *      http
	 *          // ...
	 *          .anonymous.key("key")
	 *          .authorities("ROLE_ANONYMOUS");
	 *      return http.build();
	 *  }
	 * </pre>
	 */
```